deployment/README.md

```diff
@@ -160,29 +160,29 @@ foo@bar:~$ aws emr add-steps \
+> **Note:** [Azure](https://azure.microsoft.com/en-us/services/databricks/) and [AWS](https://databricks.com/aws) Databricks is Linux-based. Therefore, if you are interested in deploying your app to Databricks, make sure your app is .NET Standard compatible and that you use [.NET Core compiler](https://dotnet.microsoft.com/download) to compile your app.
 
 Databricks allows you to submit Spark .NET apps to an existing active cluster or create a new cluster everytime you launch a job. This requires the **Microsoft.Spark.Worker** to be installed **first** before you submit a Spark .NET app.
 
 ### Deploy Microsoft.Spark.Worker
 *Note that this step is required only once*
 
-  1. Download **[db-init.sh](../deployment/db-init.sh)** and **[install-worker.sh](../deployment/install-worker.sh)** onto your local machine
-  2. Modify **db-init.sh** appropriately to point to the Microsoft.Spark.Worker release you want to download and install on your cluster
-  3. Download and install [Databricks CLI](https://docs.databricks.com/user-guide/dev-tools/databricks-cli.html)
-  4. [Setup authentication](https://docs.databricks.com/user-guide/dev-tools/databricks-cli.html#set-up-authentication) details for the Databricks CLI appropriately
-  5. Upload the files you downloaded and modified to your Databricks cluster
-     ```
-     cd <path-to-db-init-and-install-worker>
-     databricks fs cp db-init.sh dbfs:/spark-dotnet/db-init.sh
-     databricks fs cp install-worker.sh dbfs:/spark-dotnet/install-worker.sh
-     ```
-  6. Go to to your Databricks cluster homepage -> Clusters (on the left-side menu) -> Create Cluster
-  7. After configuring the cluster appropriately, set the init script (see the image below) and create the cluster.
-     
-     <img src="../docs/img/deployment-databricks-init-script.PNG" alt="ScriptActionImage" width="600"/>
-     
-  > Note: If everything went well, your cluster creation should have been successful. You can check this by clicking on the cluster -> Event Logs.
+   1. Download **[db-init.sh](../deployment/db-init.sh)** and **[install-worker.sh](../deployment/install-worker.sh)** onto your local machine
+   2. Modify **db-init.sh** appropriately to point to the Microsoft.Spark.Worker release you want to download and install on your cluster
+   3. Download and install [Databricks CLI](https://docs.databricks.com/user-guide/dev-tools/databricks-cli.html)
+   4. [Setup authentication](https://docs.databricks.com/user-guide/dev-tools/databricks-cli.html#set-up-authentication) details for the Databricks CLI appropriately
+   5. Upload the files you downloaded and modified to your Databricks cluster
+      ```shell
+      cd <path-to-db-init-and-install-worker>
+      databricks fs cp db-init.sh dbfs:/databricks/init/db-init.sh
+      databricks fs cp install-worker.sh dbfs:/spark-dotnet/install-worker.sh
```

Comment:
any reason one is spark-dotnet and another is databricks?

Reply:
db-init should be in `dbfs:/databricks/init` .  Any scripts in that folder will be run automatically for all clusters.   install-worker.sh can be in any other folder.  It can't reside in `dbfs:/databricks/init` because the script requires arguments and all scripts in `dbfs:/databricks/init` are just run as is.

Reply:
ah ok. so this is actually a fix.

Reply:
I believe so.  I previously tried using the original instructions by setting a custom init script path during [cluster creation](https://github.com/dotnet/spark/blob/master/docs/img/deployment-databricks-init-script.PNG)  , but that didn't seem to run on the worker nodes, even with cluster restarts.  Moving db-init.sh to `dbfs:/databricsks/init` fixed it.

Reply:
I'll try one more time using the cluster-scoped init script from the original instructions and let you know.

Reply:
Hmm using the cluster-scoped init scripts seems to work fine.  Reverting back.